scripts/setup

```diff
@@ -23,6 +23,10 @@ then
         mkdir -p "${HOME}/.aws"
         mkdir -p "nginx/srv/dist/"
 
+        AWS_PROFILE=${AWS_PROFILE:-raster-foundry} \
```

Comment:
Can we eliminate this and keep the call to `bootstrap` in the `Vagrantfile` without any options? `AWS_PROFILE` is already set there, and there doesn't seem to be a reason why building containers, getting the `.env` file, and building `.sbtopts` can't all happen there.

The operations in `bootstrap` would only get applied within the context of the VM, but I think that is OK now because we aren't trying to interact with the local Git configuration.

Reply:
Sure.

I had kept this call in, because I liked the idea of having the `.env` and `.sbtopts` file available on the host, although it makes more sense to have a clean file system on the host, and make those modifications within the context of the VM.